dashboard/lib/policies/lti.rb

```diff
@@ -162,6 +162,11 @@ def self.find_platform_by_issuer(issuer)
     LMS_PLATFORMS.values.find {|platform| platform[:issuer] == issuer}
   end
 
+  def self.find_platform_name_by_issuer(issuer)
+    platform_name, _ = LMS_PLATFORMS.find {|_, platform| platform[:issuer] == issuer}
+    platform_name.to_s
```

Comment:
not: This will throw if the platform can't be found, right? Don't think we guard against this

Reply:
Could do like &.to_s?

Reply:
I think this is ok as is? Calling `.to_s` on nil returns an empty string.

```
irb(main):001> nil.to_s
=> ""
```

This is also handled in the unit test.